students/brandon_nguyen/session09/mailroom_oo/donor_model.py

```diff
@@ -51,15 +68,47 @@ def __str__(self):
 
 
 class DonorCollection:
-    # Revew mailroom 1 instruction to do the following
+    """ This class will hold:
+            -all Donor objects
+            -methods to add new donor, search for a given donor
+            - extract donor
+            -create report
+            -send thank you letters to all
+    """
+    def __init__(self):
+        self.__donor_db = {}
     # We need data structure and other constants for printing
 
-    # method to update donation
-
-    # method to return donor in list getName?
+    @property
+    def donor_db(self):
+        return self.__donor_db
```

Comment:
This (exposing the internal raw data store as is) is not very desirable in object-oriented design. Any needed operations on the internal raw data better be supported through methods, not by exposing it fully like this. Besides, I don't see any place where this `donor_db` property is accessed, so I'm wondering if this is needed in the first place?

Reply:
I used it in one of my test to learn more.  I suppose I can comment it out.